### PR TITLE
Add 1.9.9: Goerli + Mainnet

### PR DESCRIPTION
**Pull request by brianmcmichael**
* Add ADAI D3M (mainnet)
* Add LERP_FAB and JOIN_FAB (mainnet and goerli)
* Bump version to 1.9.9
* Add associated ABIs
* Merge https://github.com/makerdao/mcd-changelog/pull/152 first

### CODE REVIEW

**Comment by kmbarry1** [comment]
Full pass complete; didn't find any issues beyond what Dumitru highlighted. Will look again once updates have been done.

**Comment by talbaneth** [comment]
Same as @kmbarry1

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Go to https://changelog.makerdao.com/ to see the details.
 
 1. The Protocol Engineering Core Unit creates a PR; review done by someone else from the PECU and someone from the TechOps Team.
 
-2. New contract addresses are published first to specific version URLs, e.g. https://changelog.makerdao.com/releases/mainnet/1.1.1/contracts.json . On the main page there is a disclaimer, saying that the addresses might not be yet in use.
+2. New contract addresses are published first to specific version URLs, e.g. https://changelog.makerdao.com/releases/mainnet/1.9.9/contracts.json . On the main page there is a disclaimer, saying that the addresses might not be yet in use.
```
[the remark this answers is not in the document]
Comment (brianmcmichael) [reply README.md:9]: It doesn't hurt to keep documentation up to date though.

File: releases/mainnet/1.9.9/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.9.9/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmSpJWWq73NEPBMw8pNk1GqpwYqAXnapCm3Xmy4EjSmzuh">Contract addresses (IPFS hosted)</a></li>
+				<li><a href="/releases/mainnet/1.9.9/abi/index.html">ABIs</a></li>
+
+				<li>Links to GitHub repositories
+					<ul>
+						<li>DSS deploy scripts (<a href="https://github.com/makerdao/dss-deploy-scripts/releases/tag/1.9.9" target="_blank">changelog</a>)</li>
```
[the remark this answers is not in the document]
Comment (brianmcmichael) [reply releases/mainnet/1.9.9/index.html:18]: I've done this, but I'm not sure why. The whole point of the pr is to update for this release.

[the remark this answers is not in the document]
Comment (brianmcmichael) [reply releases/mainnet/1.9.9/index.html:18]: Oh, this is for deploy scripts, nevermind.

**Comment by brianmcmichael** [comment]
Issues should be resolved here.

File: index.html [hunk 3]
```diff
@@ -97,6 +98,7 @@ <h2>Release Calendar</h2>
 				</td>
 				<td>
 					<ul>
+						<li><a href="/releases/goerli/1.9.9/index.html">Release 1.9.9</a> (Wednesday, 2021-10-27)</li>
 						<li><a href="/releases/goerli/1.9.7/index.html">Release 1.9.7</a> (Friday, 2021-10-08)</li>
```
Comment (julienmartinlevrai) [inline index.html:102]: Did we miss 1.9.8 ?

**Review by zdumitru — APPROVED** [review]

File: index.html (hunk 3 above)
Comment (kmbarry1) [reply index.html:102]: There was never a 1.9.8 release on Goerli AFAIU, and we skipped from 1.9.7 to 1.9.9 to keep consistency with mainnet.

**Review by kmbarry1 — APPROVED** [review]

**Review by talbaneth — APPROVED** [review]

**Pull request merged**